### PR TITLE
fix(plugin-meetings): add missing mqe and fix packet loss values

### PR DESCRIPTION
**Pull request by arun3528**
Fixes #https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-194861

Makes sure the packet loss info is correct compared to the server side and the jitter and other information are present 
---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mwegman — APPROVED** [review]

File: packages/node_modules/@webex/plugin-meetings/src/metrics/index.js [hunk 1]
```diff
@@ -206,9 +200,11 @@ class Metrics {
         userAgent: this.userAgentToString(),
         clientInfo: {
           clientType: options.clientType, // TODO: Only clientType: 'TEAMS_CLIENT' is whitelisted
-          os: 'linux' || this.osName(), // TODO: For some reason Mqa does not accept mac
-          osVersion: bowser.osVersion || 'unknown',
-          subClientType: options.subClientType
+          os: bowser.osname || 'linux', // TODO: For some reason Mqa does not accept mac
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/metrics/index.js:203]: TODO still needed here? What happens if `bowser.osname` is `mac`?

**Review by adamweeks — CHANGES_REQUESTED** [review]
Browser name calculation needs to be fixed by either removing the TODO statement or fixing the TODO.

File: packages/node_modules/@webex/plugin-meetings/src/metrics/index.js (hunk 1 above)
Comment (arun3528) [reply packages/node_modules/@webex/plugin-meetings/src/metrics/index.js:203]: i do see mac now `mac 11.2.0` There was a bug i fixed in the PR
bowser was returning macOs which server did not like

File: packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.js [hunk 2]
```diff
@@ -69,16 +88,27 @@ export const getVideoReceiverMqa = ({
...
+  // calculate this values
 
-  videoReceiver.common.rtpHopByHopLost = totalPacketLoss; // TODO: calculate the value
-  videoReceiver.streams[0].common.rtpJitter = statsResults[mediaType][sendrecvType].jitter * 1000 || 0;
-  videoReceiver.streams[0].common.rtpEndToEndLost = totalPacketLoss;// TODO: calculate the value
+  videoReceiver.common.maxRemoteJitter = max(statsResults[mediaType][sendrecvType].meanRemoteJitter) * 1000 || 0;
+  videoReceiver.common.meanRemoteJitter = mean(statsResults[mediaType][sendrecvType].meanRemoteJitter) * 1000 || 0;
+
+  videoReceiver.streams[0].common.rtpJitter = videoReceiver.common.maxRemoteJitter;
+  // videoReceiver.streams[0].common.rtpJitter = (statsResults.resolutions[mediaType][sendrecvType].jitterBufferDelay - lastMqaDataSent.resolutions[mediaType][sendrecvType].jitterBufferDelay) / (statsResults.resolutions[mediaType][sendrecvType].jitterBufferEmittedCount - lastMqaDataSent.resolutions[mediaType][sendrecvType].jitterBufferEmittedCount) * 1000 || 0;
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/mqaUtil.js:104]: Does this need to be here?

**Review by adamweeks — APPROVED** [review]
Looks good! I like switching more over to constants for the MQA work, will help not having magic strings.
Let one comment on a leftover comment, not a blocker though.

**Pull request merged**